lib/index.js

```diff
@@ -160,6 +160,9 @@ class ElectronDownloader {
   }
 
   get verifyChecksumNeeded () {
+    if (this.opts.disableChecksumSafetyCheck) {
+      return false
+    }
```

Comment:
If you're going to have a separate if-statement for this, I'd like to see a `console.warn` statement :smile:

Reply:
I initially had one, then realized no one would ever really see it and all that would happen is we'd get an issue asking to disable the console warning 😆

Reply:
IMO either we add a `debug()` statement to that effect, or we change the method to:

```javascript
return !this.opts.disableChecksumSafetyCheck && semver.gte(this.version, '1.3.2')
```